guardian/recipes-backend
# Correctly validate mise-en-place curation data

## What does this change?

Updates our internal data models to ensure that real data from the current version of MEP can be validated

## How to test

- Unit-testing is provided against a front downloaded on 22nd Aug 2024
- To test "in the wild", deploy to CODE
- Monitor the logs in ELK using the key `app.keyword: recipes-backend-rest-endpoints`
- Filter to `WARN` logs (or filter _out_ `DEBUG` and `INFO`)
- Upload a curation file from mise-en-place to the POST endpoint.
- You should no longer see the `WARN` logs about the format not being correct (if you expand the ELK view to 30 days you should see plenty of them)

## How can we measure success?

Able to manipulate the data, so we can apply filtering

## Have we considered potential risks?

- The Facia models will have to be updated to correspond to these
